Vecherskiys/07/Task7.4.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html> 
+<HTML> 
+    <HEAD> 
+        <META charset="utf-8"> 
+        <TITLE>Ссылки. Оформление</TITLE> 
+		<style type="text/css">
+            table {border-collapse: collapse;}
+            td {
+                width: 50px;
+                height: 50px;
+                border: 1px solid black;
+                text-align: center;
+                font-size: 35px;
+            }
+            tr:nth-child(odd) td:nth-child(even){background-color: black; color: white;}
```

Comment:
here also can be short (combine selectors by comma)

Reply:
Ok. Complete:)